libs/wingc/src/lsp/hover.rs

```diff
@@ -427,4 +427,133 @@ Json.stringify({});
       //^
   "#,
 	);
+
+	test_hover_list!(
+		class_doc,
+		r#"
+  /// Class doc
+  class Foo {
+      //^
+  }
+  "#,
+	);
+
+	test_hover_list!(
```

Comment:
These tests are great, but we should probably have one or two docstrings in our E2E tests as well. Just in case...

Reply:
done